app/components/TimeSelector/TimeSelectorHeader.js

```diff
@@ -33,18 +33,14 @@ class TimeSelectorHeader extends Component {
 
 const styles = StyleSheet.create({
   container: {
-    alignItems: 'center',
-    backgroundColor: Colors.background,
-    display: 'flex',
-    flexDirection: 'row',
-    justifyContent: 'center',
-    borderBottomColor: Colors.border,
-    borderBottomWidth: Spacing.border,
   },
   imageContainer: {
     width: 400,
     height: 100,
   },
+  titleContainer: {
+    ...Typography.screenHeader,
```

Comment:
Potentially same comment as above here.. although if it's very specific to just the screen header area I'm ok with keeping this name